.github/workflows/super-linter.yml

```diff
@@ -17,6 +17,8 @@ name: Super Linter
 on:
   pull_request:
     branches: [main]
+  push:
+    branches-ignore: [main]
```

Comment:
I’m curious why this was added. Was this just attempt to fix the failing status checks? I’d like to revert this change before merging to avoid unnecessary workflow runs.

Reply:
👋  @parkerbxyz it was to test the failing workflow run. The linter is working on my fork, something is happening with the linter on fork PRs...

I will revert 9d07ff9

Reply:
Reverted in 143406f 👍